content/articles/whois-privacy.markdown

```diff
@@ -139,3 +139,10 @@ While we will send an email reminder when the WHOIS Privacy on a domain is due t
     ![Renew WHOIS Privacy](/files/whoisprivacy-renew-page.png)
 
 </div>
+
+## WHOIS Privacy and GDPR
```

Comment:
Please add one-line after the header. Some markdown parsers will fail to properly render the text otherwise.

Reply:
Done.